README.md

```diff
@@ -16,6 +16,10 @@ This module is referencing the beta 3 build of Orchard Core ([`1.0.0-beta3-71077
 
 Define main hostname to redirect all domain variations and force SSL.
 
+#### Import
```

Comment:
Think this should be within the "Redirects" section, not the "Hostname Redirects" section.

Reply:
@peterkeating oops, quite right. I see it also mentions 'hostname redirects'.

Pushed up a commit fixing that.